docs/calculating-effective-entry-date.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: page
+title: Calculating Effective Entry Date 
+hide_hero: true
+show_sidebar: false
+menubar: docs-menu
+---
+
+# Effective Entry Date Field Constraints
+
+## 1. Transmission Windows for Same-Day ACH
+
+There are three transmission deadlines:
+
+* **1st Cutoff**: 10:30 a.m. ET
+* **2nd Cutoff**: 2:45 p.m. ET
+* **3rd Cutoff**: 4:45 p.m. ET
+
+Files must be submitted before the last cutoff time to be eligible for same-day processing.
+
+If a file is submitted after the last cutoff, entries that list the current banking day as their Effective Entry Date will be settled on the next banking day.
+
+## 2. Same-Day ACH Transaction Limit
+
+* The per-entry limit for same-day ACH is **$1,000,000**
+* Entries exceeding this amount are not eligible for same-day settlement
+* In a batch containing entries over the limit, those entries will have their Effective Entry Date adjusted to the next banking day and remaining items will still be eligible for same-day settlement
+
+## 3. Credit Entries
+
+For credit entries, the Effective Entry Date must be either:
+* The **same banking day** as the processing date for **Same Day Entries**, or
+* **One or two banking days** following the processing date for other entries
+
+## 4. Debit Entries
+
+For debit entries, the Effective Entry Date must be either:
+* The **same banking day** as the processing date for **Same Day Entries**, or
+* **One banking day** following the processing date for other entries
+
+**Note:** Receiving banks may place holds on ACH debits per their funds availability policies, separate from these settlement times.
+
+## 5. Effective Entry Date - Edge Cases
+
+### ENR Entries
+* For Automated Enrollment Entries (ENR), the Effective Entry Date field must be **space-filled**
+* ENR entries cannot be Same Day Entries
+
+### Notifications of Change
+* The ACH Operator does **not edit** the Effective Entry Date field for Notifications of Change
+
+### Return Entries
+* The ACH Operator may verify that the Effective Entry Date is properly formatted and may replace the field's existing content with the current processing date if the Effective Entry Date is invalid
+
+# Calculating Effective Entry Dates Using moov-io/base Functions
+
+## Overview
+
+The moov-io/base package provides functionality for calculating valid Effective Entry Dates when creating ACH batch headers, ensuring compliance with NACHA rules and Federal Reserve processing schedules.
```

Comment:
Nacha prefers only the N to be capitalized. Can you update that everywhere? 

```suggestion
The moov-io/base package provides functionality for calculating valid Effective Entry Dates when creating ACH batch headers, ensuring compliance with Nacha rules and Federal Reserve processing schedules.
```

Reply:
Updated "Nacha" capitalization throughout the file.